guide/installation-and-configuration/pre-installation-checklist/verify-signatures.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Verifying package signatures
+published: true
+sorting: 40
+tags: [getting started, installation]
+---
+
+On the [Download CFEngine][d], you will find sha256 checksums of all downloadable
+files which you can verify by using `sha256sum` tool.
+
+[d]: http://cfengine.com/product/free-download
```

Comment:
Instead of placing these references here inline, we have this _references file over in the documentation generator.

https://github.com/cfengine/documentation-generator/blob/master/_references.md#L20

There is already a reference to the download page, just use `[enterprise software download page]`